docker-compose.yml

```diff
@@ -65,15 +65,18 @@ services:
 
   webgui:
     container_name: webgui
-    image: pacefactory/safety-gui2-js:latest-master
+    # TODO: Revert to latest-master upon merge into main
+    image: pacefactory/safety-gui2-js:frontend-ui-config
```

Comment:
We'll want to revert this once webgui's `frontend-ui-config` is merged into `master`.